dashboard/lib/tasks/seed.rake

```diff
@@ -408,5 +408,5 @@ namespace :seed do
   timed_task incremental: [:videos, :concepts, :scripts_incremental, :callouts, :school_districts, :schools, :secret_words, :secret_pictures, :courses, :ap_school_codes, :ap_cs_offerings, :ib_school_codes, :ib_cs_offerings, :state_cs_offerings, :donors, :donor_schools, :foorm_libraries, :foorm_forms, :standards]
 
   desc "seed only dashboard data required for tests"
-  timed_task test: [:videos, :games, :concepts, :secret_words, :secret_pictures, :school_districts, :schools, :standards]
+  timed_task test: [:videos, :games, :concepts, :secret_words, :secret_pictures, :school_districts, :schools, :standards, :foorm_forms, :foorm_libraries]
```

Comment:
Now that you're using factories in all of the tests, do we still need to seed forms and libraries as part of the `seed:test` task?

Reply:
yes, the seeding is in itself the test for the forms and libraries at the moment--it tries to seed those files and will throw an error if it can't (ex. a json parsing error). This will prevent us from blocking the build on bad json. The factories are for simpler, constant versions of the forms so we have something static to test against.

Reply:
Oh right! I should have remembered that.  Thanks!